backend/src/app.module.ts

```diff
@@ -34,8 +37,10 @@ import { UserModule } from './user/user.module';
     HealthModule,
     AuthModule,
     UserModule,
+    StoryModule,
+    TestModule
   ],
-  controllers: [AppController, HealthController],
+  controllers: [AppController, HealthController, StoryController],
```

Comment:
StoryController mora it v StoryModule

Reply:
Sem tretiral app kot nek glaven module tako kot pri angularju.